src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -920,4 +921,224 @@ public void cannotCheckOutWhenItemIsNotLoanable()
       hasItemBarcodeParameter(nod),
       hasLoanPolicyParameters(notLoanablePolicy))));
   }
+
+  @Test
+  public void canCheckOutWhenItemLimitOfLoanPolicyIsReachedButMaterialTypeAndLoanTypeMissing()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    useFallbackPolicies(
+      prepareLoanPolicyWithItemLimit(1).getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.inactiveNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandard().getId(),
+      lostItemFeePoliciesFixture.facultyStandard().getId());
+
+    InventoryItemResource firstItem = itemsFixture.basedUponNod();
+    InventoryItemResource secondItem = itemsFixture.basedUponDunkirk();
+    IndividualResource steve = usersFixture.steve();
+
+    loansFixture.checkOutByBarcode(firstItem, steve);
+    loansFixture.checkOutByBarcode(secondItem, steve);
+  }
+
+  @Test
+  public void canCheckOutVideoMaterialWhenItemLimitIsReachedForBookMaterialType()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final String loanPolicyWithItemLimitId = prepareLoanPolicyWithItemLimit(1).getId().toString();
+    final String loanPolicyWithoutItemLimitId = prepareLoanPolicyWithoutItemLimit().getId().toString();
+    final String anyRequestPolicy = requestPoliciesFixture.allowAllRequestPolicy().getId().toString();
+    final String anyNoticePolicy = noticePoliciesFixture.activeNotice().getId().toString();
+    final String anyOverdueFinePolicy = overdueFinePoliciesFixture.facultyStandard().getId().toString();
+    final String anyLostItemFeePolicy = lostItemFeePoliciesFixture.facultyStandard().getId().toString();
+    final UUID book = materialTypesFixture.book().getId();
+
+    String rules = String.join("\n",
+      "priority: t, s, c, b, a, m, g",
+      "fallback-policy: l " + loanPolicyWithoutItemLimitId + " r " + anyRequestPolicy + " n " + anyNoticePolicy + " o " + anyOverdueFinePolicy + " i " + anyLostItemFeePolicy,
+      "m " + book + " : l " + loanPolicyWithItemLimitId + " r " + anyRequestPolicy + " n " + anyNoticePolicy  + " o " + anyOverdueFinePolicy + " i " + anyLostItemFeePolicy);
```

Comment:
Can this be extracted and parameterised, so that the circulation rules format isn't hard coded in many of the tests

Reply:
done